runbot_travis2docker/models/runbot_branch.py

```diff
@@ -0,0 +1,168 @@
+# © 2015 Vauxoo
+#   Coded by: moylop260@vauxoo.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import re
+import os
+
+import requests
+import subprocess
+
+from odoo import fields, models, api, tools
+
+
+class RunbotBranch(models.Model):
+    _inherit = "runbot.branch"
+
+    uses_weblate = fields.Boolean(help='Synchronize with Weblate')
+    name_weblate = fields.Char(compute='_compute_name_weblate', store=True)
+
+    @api.multi
+    @api.depends('repo_id.name', 'branch_name', 'uses_weblate')
+    def _compute_name_weblate(self):
+        for branch in self:
```

Comment:
no short circuit if `uses_weblate` is false?

Reply:
Good catch!
Thank you